supplier_app/tests/test_view.py

```diff
@@ -1058,18 +1058,24 @@ def test_redirect_to_ap_home_when_approve_a_supplier(self):
             reverse(
                 self.approve_url,
                 kwargs=self.kwargs
-            )
+            ),
+            {
+                "action": "approve",
+            }
         )
 
         self.assertEqual(response.status_code, 302)
         self.assertEqual(response.url, '/suppliersite/ap')
 
-    def test_change_taxpayer_status_to_ACTIVE_when_clicking_aprove_button(self):
+    def test_change_taxpayer_status_to_active_when_clicking_aprove_button(self):
         self.client.post(
             reverse(
                 self.approve_url,
                 kwargs=self.kwargs
-            )
+            ),
+            {
+                "action": "approve",
+            }
```

Comment:
This is pretty similar to previous test, should we add a `setUp` function?

Reply:
yes i think so, i will take care of it

Reply:
resolved